notebooks/omex.ipynb

```diff
@@ -1,16 +1,45 @@
 {
  "cells": [
   {
+   "attachments": {},
    "cell_type": "markdown",
-   "id": "2e806b7a",
+   "id": "bad57316",
+   "metadata": {},
+   "source": [
+    "[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/combine-org/combine-notebooks/main?labpath=notebooks%2Fomex.ipynb)\n",
+    "<a href=\"https://colab.research.google.com/github/combine-org/combine-notebooks/blob/main/notebooks/omex.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\\\"/></a>"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "8c0154eb",
+   "metadata": {},
+   "source": [
+    "# Simple OMEX example"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "275bf1bb",
+   "metadata": {},
+   "source": [
+    "This notebook an example for creating a COMBINE archive. A COMBINE archive is a single file bundling the various documents necessary for a modelling and simulation project, and all relevant information. The archive is encoded using the Open Modeling EXchange format (OMEX)."
```

Comment:
I would be a bit careful about using the phrase an archive is a single file

Reply:
Corrected!